app/common/services/endpoints/tag.js

```diff
@@ -41,7 +41,9 @@ function (
 
     TagEndpoint.getFresh = function (params) {
         cache.remove(Util.apiUrl('/tags/' + params.id));
-        return TagEndpoint.get(params);
+        return TagEndpoint.get(params).$promise.then(function (result) {
```

Comment:
@willdoran I had to make this change to get hold of the category.role-values in /settings/categories/edit.controller.js correctly. I don't think it affects something else, but not 100% sure?